how-to-guides/celestia-app-security.md

```diff
@@ -0,0 +1,282 @@
+---
+description: Security Guide for Celestia Nodes
+---
+
+# Security for Celestia App
+Validators in Celestia often rely on cloud or baremetal servers for their infrastructure. While some providers offer basic protection, most setups lack advanced firewall solutions such as DDoS mitigation. This makes it critical to implement your own measures. By using [FireHOL](https://github.com/firehol/firehol), we introduce a flexible and lightweight firewall that supports advanced features like rate limiting and dynamic IP blacklisting. This solution ensures that your node can handle increasing traffic while protecting it from malicious actors.
+  
+
+### Key features we’ll leverage include:
+
+  
+
+1. Traffic filtering and rate limiting to mitigate potential DDoS attacks.
+
+2. Dynamic IP blocking using blacklists like [Firehol Level 4](https://iplists.firehol.org/?ipset=firehol_level4) and [Emerging Threats](https://iplists.firehol.org/?ipset=et_block) to automatically block known malicious IPs.
+
+3. Flexible firewall rules that allow only necessary traffic for Celestia node operations.
+
+  
+
+### Install FireHOL
+
+First, ensure FireHOL is installed on your server. You can install it using the following commands:
+
+```sh
+sudo apt update
+sudo apt install firehol
+```
+
+  
+Check that FireHOL is installed and working by running:
+
+```sh
+sudo firehol version
+```
+
+  
+
+## Configure FireHOL for Celestia Node
```

Comment:
is this for "celestia-node" or a "Celestia consensus node"?

Reply:
This is for Celestia consensus node